File: DTO/Cache/Models/PersonModel.cs
using System;

namespace DTO.Cache.Models {

	public class PersonModel {

		public int PersonId { get; set; }
		public string FirstName { get; set; }
		public string LastName { get; set; }
		public char Gender { get; set; }
		public DateTime? Birthday { get; set; }
		public string IdCardNum { get; set; }
		public string Email { get; set; }
		public string PicturePath { get; set; }
		public bool IsActive { get; set; }
	}
}

File: AngularClient/src/app/ts/classes/models/person.model.ts
import { PersonI } from '../../interfaces/model.interfaces';

export class Person implements PersonI {
	public personId: number
	public firstName: string
	public lastName: string
	public gender: string
	public birthday: Date
	public idCardNum: string
	public email: string
	public picturePath: string
	public isActive: boolean
	
	constructor() { }
	
	public assignAllData(data: Object): PersonI {
		try {
			return Object.assign(this, data)
		} catch (e) {
			console.error("Can't assign data at: ", this)
			console.error("Data: ", data)
			console.error(e)
			return null
		}
	}
}
